root_locals.tf

```diff
@@ -105,4 +105,6 @@ locals {
   tmp_directory         = "/tmp"
   consignment_user_name = "consignment_api_user"
 
+  //feature access blocks
+  block_http4s = local.environment == "prod" ? true : false
```

Comment:
To be on the safe side, should probably block on `staging` as well. Strictly speaking `staging` and `prod` should be kept insync as far as possible.